src/components/atoms/OcContextualHelper/OcContextualHelper.vue

```diff
@@ -3,14 +3,20 @@
     <oc-button :id="buttonId" appearance="raw">
       <oc-icon name="question" fill-type="line" size="small" />
     </oc-button>
-    <oc-drop :drop-id="dropId" :toggle="toggleId" mode="click" close-on-click>
+    <oc-drop class="oc-width-1-1" :drop-id="dropId" :toggle="toggleId" mode="click" close-on-click>
       <div class="info-drop-content">
+        <div v-if="title" class="oc-flex oc-flex-between info-title oc-border-b oc-pb-s">
+          <h1 class="oc-m-rm" v-text="title" />
```

Comment:
I think we can't use an `h1` here. In general pretty difficult, because we can't define the `h`-level on a component level. We either have to define it as a prop or have to use an entirely different html tag for the title. Always depends on the context where the component gets added into the DOM of the page. Please read https://www.w3.org/WAI/tutorials/page-structure/headings/

Reply:
Ok for me, will use a span